### PR TITLE
Update activity-reports.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated document based on the change posted via MC275344 (Going live sep1 )

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6145

File: microsoft-365/admin/activity-reports/activity-reports.md [hunk 1]
```diff
@@ -144,19 +144,22 @@ There are circumstances where new users show up as **unknown**. This is usually
...
-1. In the admin center, go to the **Settings** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=2053743" target="_blank">Services &amp; add-ins</a> page.
+Your user list will look like this:
+ 
+![Reports - anonymized user list.](../../media/2ed99bce-4978-4ee3-9ea2-4a8db26eef02.png)
 
-2. Select **Reports**. 
-  
-3. In the **Reports** pane, select the options you want, and then save your changes.
+
+If you want to unhide user level information when you're generating your reports, a global administrator can  quickly make that change in the admin center.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/activity-reports.md]:
```suggestion
If you want to unhide user-level information when you're generating your reports, a global administrator can quickly make that change in the admin center.
```

File: microsoft-365/admin/activity-reports/activity-reports.md [hunk 2]
```diff
@@ -144,19 +144,22 @@ There are circumstances where new users show up as **unknown**. This is usually
...
   
-Your user list will look like this:
+1. In the admin center, go to the **Settings** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=2053743" target="_blank">Services &amp; add-ins</a> page.
+
+2. Select **Org Settings** and choose **Services** . 
   
-![Reports - anonymized user list.](../../media/2ed99bce-4978-4ee3-9ea2-4a8db26eef02.png)
+3. In the **Reports** pane, select the **Choose how to show user information** and  select **Show identifiable user information in reports**
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/activity-reports.md]:
```suggestion
3. In the **Reports** pane, select **Choose how to show user information** and  choose **Show identifiable user information in reports**.
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba, kindly just commit the minor suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**